src/pkcs15init/pkcs15-openpgp.c

```diff
@@ -317,17 +316,15 @@ static int openpgp_generate_key_ec(sc_card_t *card, sc_pkcs15_object_t *obj,
 			   ? SC_OPENPGP_KEYALGO_ECDH /* ECDH for slot 2 only */
 			   : SC_OPENPGP_KEYALGO_ECDSA; /* ECDSA for slot 1 and 3 */
 
-	/* extract oid the way we need to import it to OpenPGP Card */
+	/* DEE NO extract oid the way we need to import it to OpenPGP Card */
+	/* TODO DEE pass the oid. will convert to asn1 before writing */
+	/* TODO DEE not sure id test for der.len >2 is needed */
```

Comment:
Please remove the DEE identifier. Also, I do not understand what the comments try to tell me. Please clarify the comment or remove it.

Reply:
Resolving as comments are no longer in the PR